### ISSUE TITLE
[Video Plus Chat] Animate Layout Changes

### ISSUE THREAD

**Issue by LeviOrtega** [microsoft/surface-duo-compose-samples#95]
## 💡 Describe the feature

The views sample of this project has animations because of motion layout, so if we could use Constraint Layout to animate through the composable changes it would look good.

## 🆕 New component or addition to an existent one?

Addition 

## 🙋‍♀️ Do you want to build the feature?

- [ ] Yes
- [x] No

**Comment by EmmanuelDav** [comment]
@CesarValiente is this issue still open, can I handle this issue?

**Comment by khalp** [comment]
@EmmanuelDav Yes, and you can refer to the [Views sample](https://github.com/microsoft/surface-duo-window-manager-samples/tree/main/FoldingVideoPlusChat) to see the original `MotionLayout` animations mentioned in the issue description. Let us know if you have any questions, thanks!
